CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Unreleased
 ------
 
+Version 1.4.1
+------
+* Improve and simplify testing 
```

Comment:
It looks like all but one of the current changes were on testing, but I wonder if we want to mention that we started including the APIClient and OrgID in analytics when creating apps as well (we've always done it for other commands).

Reply:
I don't think testing should be mentioned in the changelog, only things that are relevant to users.